eng/performance/maui_scenarios_android.proj

```diff
@@ -4,8 +4,7 @@
 
   <PropertyGroup>
     <IncludeXHarnessCli>true</IncludeXHarnessCli>
-    <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.21566.2</MicrosoftDotNetXHarnessCLIVersion>
-    <XharnessPath>%HELIX_CORRELATION_PAYLOAD%\microsoft.dotnet.xharness.cli\$(MicrosoftDotNetXHarnessCLIVersion)\tools\net6.0\any\Microsoft.DotNet.XHarness.CLI.dll</XharnessPath>
+    <MicrosoftDotNetXHarnessCLIVersion>10.0.0-prerelease.24524.9</MicrosoftDotNetXHarnessCLIVersion>
```

Comment:
Just curious, regarding problem 3. from https://github.com/dotnet/performance/issues/4574, why cant we add a darc subscription to dotnet/xharness and let maestro update the versions?

Reply:
We spoke with Parker and the plan is to bring Android to "recent" XHarness version in this PR and fix the issues coming with that. Then a follow-up PR that will unify the Apple and Android XHarness versions and introduce the maestro auto-update.

Reply:
What Matous said is the plan. The automatic updating and unification of Apple and Android XHarness versions will be in another PR to keep each PR more focused.